src/Feather.jl

```diff
@@ -359,76 +370,53 @@ function writenulls{T <: NullableCategoricalArray}(io, A::T, null_count, len, to
     end
     return total_bytes
 end
-function writenulls(io, A::DataArray, null_count, len, total_bytes)
```

Comment:
Would it hurt keeping this for now? Would be nice to be able to convert between all array types.